.chloggen/metric-server-hostproc.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
```

Comment:
We are breaking the API, so this should be `breaking` IMO (at least I think that has been the case in opentelemetry-collector historically)

Reply:
Your comment below seems to indicate this comment is now outdated: https://github.com/open-telemetry/opentelemetry-collector/pull/7998#discussion_r1272095494

I will resolve this thread now.

Reply:
Yes, sorry about that